docs/user-guide/connectors/slack.rst

```diff
@@ -17,19 +17,24 @@ Getting Credentials
 
   1. To create the app go to https://api.slack.com/apps and click
      on **Create New App**.
+
   2. Activate the following features:
 
     - `Interactive Components <https://api.slack.com/reference/block-kit/interactive-components>`_ (if your bot uses any, e.g. buttons)
-    - Event subscriptions
-    - Bot users
-    - Permissions (for basic functionality you should subscribe to the
-      following events: ``message.channel``, ``message.groups``,
-      ``message.im`` and ``message.mpim``)
+    - Event subscriptions > Subscribe to bot events: chat.write, message.channels, message.groups, message.im & message.mpim
+    - App Home
+      - Always Show My Bot as Online
 
-  3. Get the ``Bot User OAuth Access Token`` from the OAuth & Permissions page. You will need
+  3. Get the ``Bot User OAuth Access Token`` from the OAuth & Permissions page. Click ``Install App to Workspace``
+     and allow access to your workspace. You will need
      to provide this value in your credentials later in these instructions. It should start
      with ``xoxb``.
 
+  4. The endpoint for receiving slack messages is ``http://localhost:5005/webhooks/slack/webhook``, replacing
+     the host and port with the appropriate values. This is the URL you should add in the "OAuth & Permissions" 
+     "Redirect URLs" and "Interactivity & Shortcuts"
```

Comment:
Should we make this step 2 so that we can tell them to add the URL directly when we turn on the features?

Reply:
Also, redirect URL should not be necessary

Reply:
It is required and if you delete it, the slack UI adds it back.

Reply:
Good idea about step 2. Cleaned up that up.